test/plugins/worker/lib/BlockedBy.test.js

```diff
@@ -68,14 +80,24 @@ describe('Plugin Test', () => {
             }
         };
         mockRedisConfig = {
+            connectionDetails: {
+                host: '127.0.0.1',
+                options: {
+                    password: 'test123',
+                    tls: false
+                },
+                port: 1234,
+                database: 0
+            },
             runningJobsPrefix,
             waitingJobsPrefix
         };
         helperMock = {
             updateBuildStatus: sinon.stub()
         };
 
-        mockery.registerMock('ioredis', mockRedis);
+        mockery.registerMock('ioredis', mockRedisDep);
```

Comment:
can we also add 1 test which checks if the lock acquisition fails, then what should be the output?